wordpress-mobile/WordPress-iOS
# Better voice over support in Themes cards

Fixes #8697 

To test:
1. Navigate to My Sites > Any Site > Themes
2. Activate Voice Over

The header should read "Current Theme + THEME_NAME". Each Theme card should provide a description (is it too long?) of the theme. Each card should expose the ellipsis button to voice over
